src/main/java/org/traccar/api/resource/SessionResource.java

```diff
@@ -99,9 +99,15 @@ public User get(@QueryParam("token") String token) throws SQLException, Unsuppor
     @PermitAll
     @POST
     public User add(
-            @FormParam("email") String email, @FormParam("password") String password) throws SQLException {
+            @FormParam("email") String email, @FormParam("password") String password,
+            @FormParam("code") int code) throws SQLException {
```

Comment:
Why `int` instead of `string`?

Reply:
Modified to string so when in the future other 2FA solutions are added the code can contain other characters than numbers. Currently I am only aware of numeric 2FA codes.